README.md

```diff
@@ -75,7 +75,9 @@ The following 35 microcontroller boards are currently supported:
 * [PineTime DevKit](https://www.pine64.org/pinetime/)
 * [Seeed Wio Terminal](https://www.seeedstudio.com/Wio-Terminal-p-4509.html)
 * [Seeed Seeeduino XIAO](https://www.seeedstudio.com/Seeeduino-XIAO-Arduino-Microcontroller-SAMD21-Cortex-M0+-p-4426.html)
+* [Seeed Sipeed MAix BiT](https://www.seeedstudio.com/Sipeed-MAix-BiT-for-RISC-V-AI-IoT-p-2872.html)
 * [SiFIve HiFive1](https://www.sifive.com/boards/hifive1)
+* [SparkFun Teensy 3.6](https://www.sparkfun.com/products/14057)
```

Comment:
This board is made by PJRC not by Sparkfun:

https://www.pjrc.com/store/teensy36.html

Reply:
@deadprogram 
Updated.